.gitignore

```diff
@@ -1,24 +1,14 @@
-# Don't put *.swp, *.bak, etc here; those belong in a global ~/.gitignore.
+# Don't put *.swp, *.bak, etc here; those belong in a global ~/.gitignore_global.
```

Comment:
That's the example name it uses, but the heading is "Create a global .gitignore"